governance-aepp/src/App.vue

```diff
@@ -53,45 +54,48 @@
     methods: {
       async checkAndReloadProvider() {
         if (!aeternity.address) return;
-
         const changesDetected = await aeternity.verifyAddress();
+        // Reload the page, if changes have been detected.
         if (changesDetected) this.$router.go();
       },
       async abortWalletCheck() {
         await aeternity.disableWallet();
-        this.clientAvailable = true;
+        this.foundWallet = true;
       },
       resetView() {
         this.$refs.wrapper.scrollTo(0, 0);
       }
     },
     async created() {
-
       setTimeout(() => {
         this.showSkip = true
-      }, 4000);
+      }, 5000);
 
-      // Bypass check if there is already an active wallet
-      try {
-        if (aeternity.hasActiveWallet()) {
-          return this.clientAvailable = true
-        } else if (aeternity.client && !aeternity.contract) {
-          await aeternity.initProvider();
-          return this.clientAvailable = true
-        }
+      //First try AEX-2
+      await Promise.race([
```

Comment:
If there is an active wallet we should not re-init it. We should keep the active wallet.